pymoskito/simulation_gui.py

```diff
@@ -1516,7 +1517,7 @@ def export(self, dataPoints):
             self._logger.error("Can't instantiate exporter! " + str(e))
             return
 
-        defaultPath = os.getenv('HOME')
+        defaultPath = str(Path.home())
```

Comment:
I'd rather use the current directory as default and remember the last export path in the application settings.